### PR TITLE
api-upload-image-doc

### PR DESCRIPTION
**Pull request by dungvan**

### CODE REVIEW

File: docs/api-references/uploadimage.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+###Upload Image API
```
Comment (ghost) [inline docs/api-references/uploadimage.md]: Please correct Github markdown syntax. You must preview first.

File: docs/api-references/uploadimage.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+| Key | Value |
+|---|---|
+| Accept | application/json |
+| Content-Type | image/* |
+| Authorization | {token_type} {access_token} |
+
+##### Parameter
+| Key | Description |
+|---|---|
+| file |  A binary file (up to 10Mb) |
```
Comment (ghost) [inline docs/api-references/uploadimage.md]: This parameter and parameter in sample response not same?

File: docs/api-references/uploadimage.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+}
+  
+```
+
+or
+
+```json
+{
+  "success": false,
+  "message": "cancel upload image fail"
```
Comment (ghost) [inline docs/api-references/uploadimage.md]: `cancel upload image fail` nghĩa là "Hủy bỏ upload hình thất bại"?

**Review by ghost — COMMENTED** [review]
Please check my comment.

**Comment by ghost** [comment]
Please add new doc to `resources/views/api-doc-builders/index.blade.php`

File: docs/api-references/uploadimage.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+##### Response parameter
+
+| Key | Type | Description |
+|---|---|---|
+| fileName | string | the name of image |
+
+##### Sample Response the name of file save success
+```json
+{
+  "imageName": "1505789329number-1.png"
```
Comment (ghost) [inline docs/api-references/uploadimage.md]: Should be used `snake_case`.

File: docs/api-references/uploadimage.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+}
+```
+
+**Remove image**
```
Comment (ghost) [inline docs/api-references/uploadimage.md]: **Remove image** is another API?

**Review by ghost — COMMENTED** [review]
Please check my comment

File: docs/api-references/uploadimage.md (hunk 4 above)
Comment (at-dungvan) [reply docs/api-references/uploadimage.md]: In the same api, if request has file -> upload image to server, if request has fileName -> remove the file há name = fileName on server

File: docs/api-references/uploadimage.md (hunk 5 above)
Comment (dungvan) [reply docs/api-references/uploadimage.md]: In the same api, if request has file -> upload image to server, if request has fileName -> remove the file há name = fileName on server
Reply…

File: docs/api-references/uploadimage.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+##### Sample Response
+
+```json
+{
+  "success": true
+}
+  
+```
+
+or
```
Comment (ghost) [inline docs/api-references/uploadimage.md]: We already have error response sample, you don't need to repeat again.

**Review by ghost — COMMENTED** [review]
Please check my comment

**Comment by ghost** [comment]
LGTM ⌨️

**Pull request merged**